docker/Dockerfile

```diff
@@ -8,19 +8,29 @@ ENV PATH $CASSANDRA_DIR/bin:$PATH
 
 # Untar our distribution
 ADD palantir-cassandra-*tgz /
+RUN mkdir $CASSANDRA_DIR/logs; \
+  mkdir -p $CASSANDRA_DIR/data/saved_caches; \
+  mkdir -p /var/lib/cassandra/persistent_settings; \
+  mkdir -p /var/lib/cassandra/data; \
+  mkdir -p /var/lib/cassandra/commitlog; \
+  mkdir -p /var/data/tmp; \
+  chown -R cassandra:cassandra /var/lib/cassandra /var/data/tmp $CASSANDRA_DIR
 
-# We give the cassandra user permissions to create things in the Cassandra dir
-RUN ["sh", "-c", "chmod +rwx ${CASSANDRA_DIR}"]
-RUN ["sh", "-c", "chown cassandra:cassandra ${CASSANDRA_DIR}"]
+
+# Install java 17
+RUN apt-get update && apt-get -y install openjdk-17-jdk strace
+ENV JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64/
+ENV PATH=$JAVA_HOME:$PATH
 
 # Configs set up
 ENV CASSANDRA_CONFIG $CASSANDRA_DIR/conf/
 ENV CASSANDRA_YAML $CASSANDRA_CONFIG/cassandra.yaml
 ENV CASSANDRA_ENV $CASSANDRA_CONFIG/cassandra-env.sh
 COPY cassandra.yaml $CASSANDRA_YAML
 COPY cassandra-env.sh $CASSANDRA_ENV
+COPY logback.xml $CASSANDRA_CONFIG/logback.xml
 
-ENV _JAVA_OPTIONS=-Dcassandra.skip_wait_for_gossip_to_settle=0
+ENV _JAVA_OPTIONS="-Dcassandra.skip_wait_for_gossip_to_settle=0 -Dpalantir_cassandra.is_new_cluster=true"
 
```

Comment:
why do we need to set is_new_cluster here if we weren't previously?

Reply:
The image before was completely broken :) That said, I forgot that we enable this behavior within our seed providers internally, but the OSS version if largely unaffected by this change. Happy to remove.